src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -389,9 +380,9 @@ public void It_implicitly_defines_compilation_constants_for_the_target_framework
         }
 
         [Theory]
-        [InlineData(new string[] { }, "windows", "10.0.19041.0", new[] { "WINDOWS" })]
-        [InlineData(new[] { "1.0", "1.1" }, "ios", "1.1", new[] { "IOS" })]
-        [InlineData(new[] { "11.11", "12.12", "13.13" }, "android", "12.12", new[] { "ANDROID" })]
+        [InlineData(new string[] { }, "windows", "10.0.18362.0", new[] { "WINDOWS", "WINDOWS10_0_18362_0", "WINDOWS7_0_OR_GREATER", "WINDOWS8_0_OR_GREATER", "WINDOWS10_0_17763_0_OR_GREATER", "WINDOWS10_0_18362_0_OR_GREATER" })]
```

Comment:
I think that we decided not to do compilation constants with versions for platforms.  So I don't think we should be defining `WINDOWS10_0_18362_0`, or any of the Windows "OR_GREATER" constants.

@terrajobst does that sound right?

Reply:
> The OS flavors will also gain the XXX_OR_GREATER variants
For example, net5.0-windows10.0.19041.0 will also define WINDOWS, WINDOWS10_0_19041_0, WINDOWS10_0_19041_0_OR_GREATER, etc.

This is what the spec says, I think we do want to define these.

Reply:
Looks like I was wrong about how we plan to handle these.